src/Settings.cpp

```diff
@@ -2,6 +2,7 @@
 #include <sys/stat.h>
 #include <fstream>
 #include <iostream>
+#include <giomm/file.h>
```

Comment:
You include the cpp api but use the c functions below. Please use the appropriate cpp functions like `Gio::...`